src/cmd-build

```diff
@@ -310,6 +310,13 @@ EOF
 
 echo '{ "images": {} }' > tmp/images.json
 
+overridesjson=tmp/overrides.json
+if [ -f "${overrides_active_stamp}" ]; then
+    echo '{ "coreos-assembler.overrides-active": true }' > "${overridesjson}"
```

Comment:
A good follow-up here would be to actually break down what the overrides are (which actually intersects with https://github.com/coreos/coreos-assembler/issues/740).